VimeoNetworking/VimeoNetworking/Request.swift

```diff
@@ -155,50 +130,16 @@ public struct Request<ModelType: MappableResponse>
      - returns: an initialized `Request`
      */
     public init(method: VimeoClient.Method = .GET,
-         path: String,
-         parameters: VimeoClient.RequestParameters? = nil,
-         page: Int? = nil,
-         itemsPerPage: Int? = nil,
-         modelKeyPath: String? = nil,
-         cacheFetchPolicy: CacheFetchPolicy? = nil,
-         shouldCacheResponse: Bool? = nil,
-         retryPolicy: RetryPolicy? = nil)
+                path: String,
+                parameters: AnyObject? = nil,
+                modelKeyPath: String? = nil,
+                cacheFetchPolicy: CacheFetchPolicy? = nil,
+                shouldCacheResponse: Bool? = nil,
+                retryPolicy: RetryPolicy? = nil)
     {
         self.method = method
-        
-        let (path, query) = path.splitLinkString()
         self.path = path
-        
-        if let query = query,
-            let queryParameters = query.parametersFromQueryString()
-        {
-            var page = page
-            if let pageValue = queryParameters[self.PageKey]
-            {
-                page = page ?? Int(pageValue)
-            }
-            self.page = page
```

Comment:
Does removing this limit our ability to inspect the page and items per page values on the request?